src/client/pages/Tutorials/GuidelinesAndSpecifications/GuidelinesAndSpecifications.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { AssessmentName, AssessmentNames } from 'meta/assessment'
+
+import { useUser } from 'client/store/user'
+import { useLanguage } from 'client/hooks/useLanguage'
+import AssessmentSwitch from 'client/components/AssessmentSwitch'
+import CycleSwitch from 'client/components/CycleSwitch'
+
+export const userGuideUrls: Record<string, string> = {
+  en: 'https://www.fao.org/3/cc4690en/cc4690en.pdf',
+  ar: 'https://www.fao.org/3/cc4690ar/cc4690ar.pdf',
+  ch: 'https://www.fao.org/3/cc4690zh/cc4690zh.pdf',
+  es: 'https://www.fao.org/3/cc4690es/cc4690es.pdf',
+  fr: 'https://www.fao.org/3/cc4690fr/cc4690fr.pdf',
+  ru: 'https://www.fao.org/3/cc4690ru/cc4690ru.pdf',
+}
+
+const Fra2025GuidelinesAndSpecifications: React.FC = () => {
+  const { t } = useTranslation()
+  const lang = useLanguage()
+
+  return (
+    <>
+      <div className="landing__page-header">
+        <h1 className="landing__page-title title">{t('tutorial.guidelinesAndSpecifications')}</h1>
+      </div>
+      <div className="list-tutorial ">
+        <a target="_blank" rel="noreferrer" href={userGuideUrls[lang] ?? userGuideUrls.en}>
+          {t('common.userGuide')}
+        </a>
+      </div>
+    </>
+  )
+}
+
+const FraCycleComponents: { [key: AssessmentName]: React.FC } = {
+  '2020': null,
+  '2025': Fra2025GuidelinesAndSpecifications,
+}
+
+const FraGuidelinesAndSpecificationss: React.FC = () => {
+  return <CycleSwitch components={FraCycleComponents} />
+}
+
+const AssessmentComponents: { [key: AssessmentName]: React.FC } = {
+  [AssessmentNames.fra]: FraGuidelinesAndSpecificationss,
+  [AssessmentNames.panEuropean]: null,
+}
```

Comment:
Wonder if AssessmentSwitch is redundant? Should CycleSwitcher support assessmentName?

Reply:
I tried changing from `React.FC` to `ReactNode`  as your type suggests, but it got too messy and it would have been inconsistent with the AssessmentSwitch component logic. I ended up using this type: `Record<AssessmentName, Record<CycleName, React.FC>>`.
I also removed the placeholder. It is now an optional prop. 
The components that use CycleSwitch are now easier to follow. Thanks for the suggestions!